typescript/organisations.d.ts

```diff
@@ -226,3 +227,19 @@ export type ProductNotification = NewProductNotification & {
   createdAt: string
   updatedAt: string
 }
+
+export type NewAIDisclaimer = {
+  organisationId: string
+  type: 'FORMS_BUILDER'
+  acceptedBy: UserProfile
+  updatedBy?: UserProfile
+  data: {
+    allowedFormsAppEnvironmentIds: number[]
+  }
+}
+
+export type AIDisclaimer = NewAIDisclaimer & {
+  id: number
+  createdAt: string
```

Comment:
```suggestion
  acceptedAt: string
```

Not going to die on this hill. If you would prefer to stick with `createdAt`, inherit the `IdResource` type from misc.